pages/api/auth/[...nextauth].ts

```diff
@@ -26,6 +28,26 @@ export const authOptions = {
       return session;
     },
   },
+  events: {
```

Comment:
Stripe customer is created after user is created and they are linked such that checkout is specific to the user.